domene/src/main/java/no/nav/foreldrepenger/selvbetjening/uttak/KontoBeregningDto.java

```diff
@@ -3,11 +3,15 @@
 import java.util.List;
 
 
-public record KontoBeregningDto(List<KontoDto> kontoer, Minsteretter minsteretter) {
+public record KontoBeregningDto(List<KontoDto> kontoer, Minsteretter minsteretter, Utvidelser utvidelser) {
 
+    @Deprecated(forRemoval = true)
     public record Minsteretter(int farRundtFødsel, int toTette) {
     }
 
+    public record Utvidelser(int farRundtFødsel, int toTette, int tilleggFlerbarn) {
```

Comment:
For min del så kan godt farRundtFødsel og toTette fortsatt ligge i minsterett. Er det en grunn til å legge de under "utvidelser"?